changes/1386.bugfix.md

```diff
@@ -0,0 +1,2 @@
+Remove incorrect `is_nsfw` field from threads.
```

Comment:
You should document that the parent attribute was moved from GuildChannel to PermissibleGuildChannel as well since that's the change which effects more ppl here really

Reply:
Changed :)